frontend/src/pages/start/libs/components/voice-tone-form/libs/site-tone-type-to-readable.enum.ts

```diff
@@ -0,0 +1,8 @@
+import { SiteToneType } from '~/packages/sites/libs/enums/enums.js';
```

Comment:
move to 
`frontend/src/pages/start/libs/components/voice-tone-form/libs/maps/site-tone-type-to-readable.map.ts`